articles/clients/how-to-rotate-client-secret.md

```diff
@@ -0,0 +1,50 @@
+---
+description: This page lists different ways of how to update your client's secret.
+crews: crew-2
+---
+
+# Rotate the Client Secret
+
+The client secret protects your resources by only granting tokens to requestors if they're authorized. Protect your client secrets; if any are ever compromised, you should rotate to a new one. Please remember that all authorized apps will therefore need to be updated with the new client secret.
+
+## Rotate the Client Secret in the Dashboard
+
+You can rotate your client secret under [Clients in the Dashboard](${manage_url}/#/clients). Choose the client you wish to edit by clicking on its name *or* the **Settings** gear icon associated with the client.
+
+![](/media/articles/clients/change-client-secret/clients.png)
+
+On the **Settings** page, the **Client Secret** will be the fourth parameter listed. To the right, click on the **rotation** icon to rotate your secret. You can view you your new secret by checking the box next to **Reveal client secret**.
+
+Scroll to the bottom of the Settings page, and click **Save Changes**.
+
+![](/media/articles/clients/change-client-secret/client-settings.png)
+
+## Rotate the Client Secret Using the Management API
+
+You can rotate your client's secret by making a `POST` call to the [Rotate a Client Secret endpoint](/api/management/v2#!/Clients/post_rotate_secret) of the Management API.
+
+::: note
+To make calls to the Management API, you'll need to [obtain and provide a valid access token](/api/management/v2/tokens).
+:::
+
+```har
+{
+	"method": "POST",
+	"url": "https://${account.namespace}.auth0.com/api/v2/clients/YOUR_CLIENT_ID/rotate-secret",
+	"httpVersion": "HTTP/1.1",
+	"cookies": [],
+	"headers": [{
+		"name": "Authorization",
+		"value": "Bearer MGMT_API_ACCESS_TOKEN"
+	}],
+	"queryString": [],
+	"postData": {},
+	"headersSize": -1,
+	"bodySize": -1,
+	"comment": ""
+}
+```
+
```

Comment:
Add a couple of bullets on the info the users should replace: MGMT_API_ACCESS_TOKEN, YOUR_CLIENT_ID